test/test.js

```diff
@@ -42,9 +43,7 @@ describe('Localize APIs', () => {
                 sourceLanguage: '',
             };
             localizeService.project.create(data, function (err, result) {
-                if (err) {
```

Comment:
Removed if condition because some earlier test cases were copy/pasted with correct params, since there was no error the test cases were never actually executed.